protocol/x/clob/keeper/msg_server_proposed_operations.go

```diff
@@ -17,7 +16,7 @@ func (k msgServer) ProposedOperations(
 
 	defer func() {
 		if err != nil {
-			errorlib.LogErrorWithBlockHeight(k.Keeper.Logger(ctx), err, ctx.BlockHeight(), metrics.DeliverTx)
```

Comment:
Possible we could also add a basic helper function to add success / failure metrics for each message handler?

Reply:
yeah that's a good idea. I think it would be a nice pattern to use `defer` and emit failure if `err != nil` and success otherwise. Would be pretty easy, won't do it in this PR though don't want to overload

Reply:
https://github.com/dydxprotocol/v4-chain/pull/529/files